bot.py

```diff
@@ -8,13 +8,46 @@
 from cogs.error_handler import CommandErrorHandler
 from cogs.help import GetHelp
 
+# Add in intent to allow us to monitor member stats
+
+intents = discord.Intents.default()
+intents.members = True
+
+
 bot = commands.Bot(command_prefix='$')
 
 
 @bot.event
 async def on_ready():
     print("Ready to go!")
 
+    # ---------------------------------------------------
+    # List true member count and 'online' members
+    # ---------------------------------------------------
+
+    #guild_name = "ExplorerTest"
+    guild_name = "Astra Nova Network"
```

Comment:
If `guild_name` is not redundant, then we should consider loading it in as a constant from a textfile, rather than as a hardcoded value. A bit like how the moderators are read in.